packages/react/src/themes/teams/components/Attachment/attachmentStyles.ts

```diff
@@ -64,23 +62,28 @@ const attachmentStyles: ComponentSlotStylesInput<AttachmentProps, AttachmentVari
     marginRight: v.iconSpace,
   }),
 
-  action: ({ variables: v }): ICSSInJSStyle => ({
-    flex: '0 0 auto',
-
-    [`& .${Icon.className}`]: {
-      color: v.textColor, // this breaks the color change on hover
-    },
+  action: ({ props: p, variables: v, theme: { siteVariables } }): ICSSInJSStyle => {
+    const iconFilledStyles = getIconFillOrOutlineStyles(false)
 
-    ':hover': {
-      [`& .${teamsIconClassNames.filled}`]: {
-        display: 'block',
+    return {
+      [`& .${Icon.className}`]: {
+        color: v.textColor, // this breaks the color change on hover
```

Comment:
Should we use different color on regular/hover icons? Should we add condition about this? What this actually break?

Reply:
I'm just moving existing code around, I did not actually make that change; you can see that if you follow the history carefully :)